### DISCUSSION TITLE
How to check the current language?

### DISCUSSION THREAD

**Discussion by sgon00** [helix-editor/helix#5847] [Q&A] [answered]
Hi, I am new to helix. When I open a file, I figured out I can setup the language by using `:lang rust`. But how to check what the current language selecction is? I found `:theme` can return the current theme, but `:lang` simply returns an error. I can't figure this out by checking issues/discussion/doc/googling. Thanks a lot.

**Comment by the-mikedavis** [comment, marked as answer]
You can use `:set statusline.center ["file-type"]` to temporarily add the file-type statusline element which shows the current language.

I think it would be fine to add that behavior to `:set-language`/`:lang` to show the current language if no arguments are provided.

**Reply by sgon00** [reply]
Thanks a lot for the answer. I have put

```toml
[editor.statusline]
center = ["file-type"]
```

to `config.toml`.
